src/configure.py

```diff
@@ -419,7 +435,8 @@ def onprefinal(input):
             {"name": "go", "src": "foundersmap", "dst": "ownersmap"},
             {"name": "go", "src": "ownersmap", "dst": "mindelegation"},
             {"name": "go", "src": "mindelegation", "dst": "mindelegationtarget"},
-            {"name": "go", "src": "mindelegationtarget", "dst": "exclude"},
+            {"name": "go", "src": "mindelegationtarget", "dst": "minpayment"},
```

Comment:
Am I seeing the "inheritance" here you mentioned in the docs?

Reply:
No, this part of code simply (well, not so _simple_ as it _should_ be in my opinion) sets the order of options during the configuration stage.
I explained what I meant by "inheritance" [here](https://github.com/tezos-reward-distributor-organization/tezos-reward-distributor/pull/613#discussion_r974822521).